File: seungbeom/23_01_12_01.java
class Solution {
    public String solution(int n) {
        String answer = "";
        StringBuilder answerBuilder = new StringBuilder();

        for (int i = 0; i < n/2; i++) {
            answerBuilder.append("수박");
        }

        if ( n%2 == 1) {
            answerBuilder.append("수");
        }

        return answerBuilder.toString();
    }
}

File: minseon/23_01_12_01.py
# 수박수박수박수박수박수?

def solution(n):
    melon = "수박"
    answer = ''
    if n%2 == 0 :
        answer = melon*(n//2)
    else:
        answer = melon*(n//2)+"수"
    return answer

print(solution(3))
